VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddCommentFragment.java

```diff
@@ -153,4 +154,18 @@ public void onException(Exception ex) {
         createCommentTask.setProgressDialog(createCommentProgressBar);
         createCommentTask.execute();
     }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+
+        if (getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE) {
```

Comment:
I believe its to not have the keyboard pop up immediately in landscape mode when trying to create a new post/response/comment.